src/common/core.ts

```diff
@@ -0,0 +1,112 @@
+// The MIT License (MIT)
```

Comment:
Let's make this `src/common/change.ts` (or `src/common/utilities/change.ts` after merging the v1/v2 partition PR)

Reply:
changed to `src/common/change.ts`